src/Playwright.CLI/Playwright.CLI.csproj

```diff
@@ -20,8 +20,8 @@
     <IsPackable>true</IsPackable>
   </PropertyGroup>
   <PropertyGroup>
-    <AssemblyVersion>1.0.0</AssemblyVersion>
-    <PackageVersion>$(AssemblyVersion)-alpha-5</PackageVersion>
+    <AssemblyVersion>1.0.1</AssemblyVersion>
```

Comment:
We should not land stable package versions in trunk, so this should become -alpha-6. You won't even be able to cherry-pick it like this, and for a good reason.

Reply:
Changed it to `-next-1`.